panacea/grpc_client.go

```diff
@@ -149,3 +153,37 @@ func (c GrpcClient) GetDataPassRedeemHistory(redeemer string, poolID uint64) (da
 
 	return response.GetDataPassRedeemHistories(), nil
 }
+
+func (c GrpcClient) GetDataCerts(poolID, round uint64) ([]datapooltypes.DataValidationCertificate, error) {
```

Comment:
It seems that all certs belonging to the round of pool are returned.
Our DataValidator memory can run into problems if the amount of imported certs is too large.(It is more likely to happen if there is a lot of data per round))
How about modifying it to be pagination?

Reply:
Exactly!

This query is for getting all the data certs, so it would be good to set data pipeline to handle data certs than pagination. We can improve it for future. Thank you for your comment.